docs/README_DD_api.md

```diff
@@ -0,0 +1,475 @@
+# Dimension Data Ansible Modules
+### *Local copy of files modules*
+
+---
+### Requirements
+* See official Ansible docs
+
+---
+### Modules
+
+  * [dimensiondata_backup - enable or disable backups for a host.](#dimensiondata_backup)
+  * [dimensiondata_backup client - add/delete backup client for a host](#dimensiondata_backup client)
+  * [dimensiondata_compute - create, terminate, start or stop an server in dimensiondata](#dimensiondata_compute)
+  * [dimensiondata_disk - add/remove/modify disks](#dimensiondata_disk)
+  * [dimensiondata_get_unallocated_public_ips - get specified number of free addresses; provision to reach requested number.](#dimensiondata_get_unallocated_public_ips)
```

Comment:
These are all that auto-generated.
Didn't realize some where missing.
Will check.


On Tue, Jul 12, 2016 at 11:15 AM, Aimon Bustardo <notifications@github.com>
wrote:

> In docs/README_DD_api.md
> <https://github.com/NexusIS/ansible-dimensiondata/pull/1#discussion_r70491781>
> :
>
> > @@ -0,0 +1,475 @@
> > +# Dimension Data Ansible Modules
> > +### *Local copy of files modules*
> > +
> > +---
> > +### Requirements
> > +* See official Ansible docs
> > +
> > +---
> > +### Modules
> > +
> > +  * [dimensiondata_backup - enable or disable backups for a host.](#dimensiondata_backup)
> > +  * [dimensiondata_backup client - add/delete backup client for a host](#dimensiondata_backup client)
> > +  * [dimensiondata_compute - create, terminate, start or stop an server in dimensiondata](#dimensiondata_compute)
> > +  * [dimensiondata_disk - add/remove/modify disks](#dimensiondata_disk)
> > +  * [dimensiondata_get_unallocated_public_ips - get specified number of free addresses; provision to reach requested number.](#dimensiondata_get_unallocated_public_ips)
>
> You are missing some of the modules here. Will you add them later?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/NexusIS/ansible-dimensiondata/pull/1/files/8df60955ffdbeda67007f8a82b7dcc4e23ee7d17#r70491781>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AI6_IW9j9vuT6906EDzy8J3mxHOwnqCUks5qU9nVgaJpZM4JKqRJ>
> .
>